.gitignore

```diff
@@ -29,3 +29,4 @@ _CoqProject
 /..coq_makefile_output.d
 /UniMath/.dir-locals.el
 /_build/
+*/CONTENTS.md
```

Comment:
Why did you add `CONTENTS.md` to the gitignore?

Reply:
Ah, this is a remnant of a recent change. We should probably remove `CONTENTS.md` from the makefile.